src/state/mint/hooks.ts

```diff
@@ -160,10 +160,16 @@ export function useDerivedMintInfo(
       wrappedCurrencyAmount(currencyBAmount, chainId),
     ]
     if (pair && totalSupply && tokenAmountA && tokenAmountB) {
-      return pair.getLiquidityMinted(totalSupply, tokenAmountA, tokenAmountB)
-    } else {
-      return undefined
+      try {
+        return pair.getLiquidityMinted(totalSupply, tokenAmountA, tokenAmountB)
+      } catch (error) {
+        if (error.InsufficientInputAmountError) {
```

Comment:
this is a typo. should be isInsufficientInputAmount

Reply:
good catch ,submitted a pr